src/data/builder-tools.js

```diff
@@ -802,6 +802,13 @@ export const Showcases = [
     getstarted: "/docs/operate-a-stake-pool/frankenwallet",
     tags: ["operatortool"],
   },
+  { 
+    title: "cnft", 
+    description: "A library for validating metadata for the 721 metadatum property. Complies with community standards", 
+    preview: require("./builder-tools/cnft9000.png"), 
+    website: "https://github.com/ada9000/cnft#readme", 
+    getstarted: null, 
+    tags: ["javascript", "typescript", "nft"] },
```

Comment:
```suggestion
    tags: ["javascript", "typescript", "nft"],
  },
```
Please tune up the white space to look like the previous submissions; otherwise we're almost guaranteed to get merge conflicts with concurrent PRs 🙏

Reply:
resolved